packages/graph/hash_graph/lib/graph/src/shared/identifier/time/axis.rs

```diff
@@ -27,8 +27,12 @@ pub enum TimeAxis {
     TransactionTime,
 }
 
-/// Time axis for the projected image time.
+/// Time axis for the [`Image`] time axis used in [`TimeProjection`]s.
```

Comment:
This comments read slightly strange to me, perhaps just this?
```suggestion
/// Time axis for the [`Image`] used in [`TimeProjection`]s.
```